pandas/core/indexes/datetimelike.py

```diff
@@ -121,6 +121,45 @@ def floor(self, freq):
 
     @Appender(_round_doc % "ceil")
     def ceil(self, freq):
+        """
+        Floor the index to the specified freq.
+
+        Links to the available [frequency strings](http://pandas.
+        pydata.org/pandas-docs/stable/timeseries.html#offset-aliases)
+        and [frequency objects](http://pandas.pydata.org/pandas-docs/
+        stable/timeseries.html#dateoffset-objects).
+
+        Parameters
+        ----------
+        freq : str, object
+            String or object specifying the frequency to round down to.
```

Comment:
Not sure what `object` means in this case. In the examples it seems that it can also be `None`. Could you check exactly what it accepts, and make it a bit more clear please? Didn't use this method myself, and with this description I'm still confused on what should I use for frequency.